tools/ui-components/src/drop-down-button/drop-down-button.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { Menu, Transition } from '@headlessui/react';
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ');
+}
+export function DropDownButton() {
+  return (
+    <Menu as='div' className='relative inline-block text-left'>
+      <div>
+        <Menu.Button className='inline-flex justify-center w-full rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-sm font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-100 focus:ring-indigo-500'>
+          Options
+          <FontAwesomeIcon
+            icon={faChevronDown}
+            className='mt-2 ml-2 -mr-1 h-3 w-3 text-violet-200 hover:text-violet-100'
+            aria-hidden='true'
+          />
+        </Menu.Button>
+      </div>
+
+      <Transition
+        as={Fragment}
+        enter='transition ease-out duration-100'
+        enterFrom='transform opacity-0 scale-95'
+        enterTo='transform opacity-100 scale-100'
+        leave='transition ease-in duration-75'
+        leaveFrom='transform opacity-100 scale-100'
+        leaveTo='transform opacity-0 scale-95'
+      >
```

Comment:
Just wondering, do we want to have animation as default?

There are instances like menu carrying `Sign in` and `Sign out`, campers will use it frequently and seeing animation can be obnoxious for some.

And how does tailwind interact with `reduce-motion`, does it alter the `utility classes` value?